cinema/views.py

```diff
@@ -16,7 +16,7 @@ def cinema_list(request):
     if request.method == "POST":
         serializer = MovieSerializer(data=request.data)
 
-        if serializer.is_valid():
+        if serializer.is_valid(raise_exception=True):
```

Comment:
if you use serializer.is_valid(raise_exception=True): you don't need return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST) and `if `. because serializer.is_valid(raise_exception=True) will raise 400 
 exception

Reply:
Ok got it, thank you for the explanation!👍

Reply:
Let me leave comments please in order to not forget this.